quarkus/config-api/src/main/java/org/keycloak/config/BootstrapAdminOptions.java

```diff
@@ -27,13 +33,28 @@ public class BootstrapAdminOptions {
 
     public static final Option<String> CLIENT_ID = new OptionBuilder<>("bootstrap-admin-client-id", String.class)
             .category(OptionCategory.BOOTSTRAP_ADMIN)
-            .description("Client id for the temporary bootstrap admin service account." + USED_ONLY_WHEN)
+            .description("Client id for the bootstrap admin service account." + USED_ONLY_WHEN)
             .defaultValue(DEFAULT_TEMP_ADMIN_SERVICE)
             .build();
 
     public static final Option<String> CLIENT_SECRET = new OptionBuilder<>("bootstrap-admin-client-secret", String.class)
             .category(OptionCategory.BOOTSTRAP_ADMIN)
-            .description("Client secret for the temporary bootstrap admin service account." + USED_ONLY_WHEN + NON_CLI)
+            .description("Client secret for the bootstrap admin service account." + USED_ONLY_WHEN + NON_CLI)
+            .build();
+
+    public static final Option<Boolean> IS_TEMPORARY = new OptionBuilder<>("bootstrap-admin-temporary", Boolean.class)
+            .category(OptionCategory.BOOTSTRAP_ADMIN)
+            .description("Indicates whether the admin user or service account is temporary." + USED_ONLY_WHEN + NON_CLI)
```

Comment:
Why would we suggest the CLI, it's not a secret.
```suggestion
            .description("Indicates whether the admin user or service account is temporary." + USED_ONLY_WHEN)
```